### DISCUSSION TITLE
Self-signed certificate

### DISCUSSION THREAD

**Discussion by sl8vz** [frw/react-native-ssl-public-key-pinning#215] [General]
Hi,

Do you know if using this implementation of ssl pinning would allow to use self-signed certificate ?

Thanks,
Sebastien

**Comment by frw** [comment]
Hi @sl8vz,

Apologies for the delay. Unfortunately this library does not support self-signed certs without installing the cert on the device. Please refer to the following discussion if you'd like to learn more:  https://github.com/frw/react-native-ssl-public-key-pinning/issues/89

**Comment by yalipanerga** [comment]
Hi @frw, mas orang indo ya mas ? kalau baca2 dari #89 berarti ini memang ga bisa self signed ya ? issue nya memang karena dari trustkit and okhttp nya kah ? fyi di article link okhttp yang di attach nyaa udah not found link nya mas. untuk self signed kepikiran di listen dlu di awal app error ketika appy ssl key nya. kemudian di reapply di intercept request network nya, kemudian setelah di apply di retry. possible ga yaa ?. tujuan nya sbenarnya mau menjawab issue jadi user ga perlu hardclose app nya dulu ketika renew ssl key.
